sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawServiceSpec.java

```diff
@@ -61,6 +62,14 @@ public Builder setEnv(Map<String, String> env) {
             return this;
         }
 
+        /**
+         * Set whether the rendering should be strict.
+         */
+        public Builder setStrictRendering(boolean isStrict) {
```

Comment:
How about an `enableStrictRendering()` that sets the value to `true`?

Reply:
Done.